### DISCUSSION TITLE
Disabling the floating miniplayer

### DISCUSSION THREAD

**Discussion by p0358** [PoomSmart/YTABConfig#130] [General]
Whoever designed that anti-human piece of shit needs to... I won't finish the sentence. You can't even bring up the video page back with it, it's just broken and pointless.

Anyway, you can disable it by turning off: `enableIosFloatingMiniplayer`

You're welcome

**Comment by foxnoodles** [comment]
I have no idea how you failed to notice that there are plenty pf FloatingPlayer related settings available and one of them enables swipe up to expand. This is a great feature and there is nothing wrong with its design. It literally mimics the native PiP mode.

**Reply by p0358** [reply]
> enables swipe up to expand

Whatever, it's ridiculous that it's not even enabled to begin with, but I just wanted it gone

> This is a great feature and there is nothing wrong with its design

Of course there is, it occupies half of the screen on the bottom, it's meant for you to see some kind of thumbnail of next video and have to scroll farther back to see what it is and the title, but then you see another part of thumbnail and another part of video. It's a predatory design that's meant to keep you scrolling on the app instead of cleanly finishing your scrolling.

You would be deeply mistaken if you think changes like this happen, on an app of this scale, without hidden ulterior reasoning behind them. YouTube is in its peak enshittification stage and they're looking for all possible ways to extract more money.

> It literally mimics the native PiP mode.

PiP was always literally meant to be used outside of the app, not inside of it.

**Reply by foxnoodles** [reply]
You are way off with your UX assessments. 

First of all NO it does take any more screen space than the original bottom bar miniplayer which takes an entire bottom portion of the screen. The floating miniplayer takes half of that. Also it can be resized and moved around, that setting is also available. 

Second of all I don’t see how something not enabled by default can be seen as an argument when we are literally talking about hidden A/B flags. Just go ahead and enable it. 

And last but not least PiP is meant to be used as a video player inside any app with exception of the source. So it is literally designed to be used as a window floating over whatever app you are running. So saying that it was not meant to do that is again false.

**Reply by p0358** [reply]
> Also it can be resized and moved around, that setting is also available.

Yeah, every time manually. It does not persist the position. If it did, maybe that'd be half of the problem solved.

> Second of all I don’t see how something not enabled by default can be seen as an argument when we are literally talking about hidden A/B flags. Just go ahead and enable it.

They are enabling it randomly as part of A/B by default, yes, I got it enabled by YouTube on its own, that's the whole thing. Go into app reviews, sort by newest and find a bunch of 1-stars about it.

> And last but not least PiP is meant to be used as a video player inside any app with exception of the source. So it is literally designed to be used as a window floating over whatever app you are running. So saying that it was not meant to do that is again false.

"with the exception of the source", you said it yourself. And this mini-player is used within YouTube itself, not outside of it...

There was never a problem with PiP and miniplayer being completely separated things, for separate use-cases and working in distinct ways to their own advantage. I mean it's lot like this floating miniplayer *is* PiP, it's not gonna stick around if you go to another app either.

**Comment by foxnoodles** [comment]
It saves position and size on my end. 
with expection of source means that you don need it when you are on the source page or app as it already shows the player. So you don’t seem to be want to understand what I am saying. 

Again whatever has youtube enabled by default does not apply here where we literally hacked through their settings to change them to whatever we want. 

And yes it will “stick around” if you have PiP enabled.

**Reply by p0358** [reply]
> Again whatever has youtube enabled by default does not apply here where we literally hacked through their settings to change them to whatever we want.

Sure, that's exactly why I'm sharing how to disable it here for those interested, that's it.

> And yes it will “stick around” if you have PiP enabled.

Native iOS's PiP has different dimensions and UI, so the UX of that is probably gonna be terrible anyways...

**Comment by foxnoodles** [comment]
No it is exactly the same ui and dimensions
